administrator/components/com_admin/script.php

```diff
@@ -2950,4 +2951,44 @@ protected function fixFilenameCasing()
             }
         }
     }
+
+    /**
+     * Fixes escaping of characters in language override files
+     *
+     * @return  void
+     *
+     * @since   4.4.2
```

Comment:
```suggestion
     * @since   __DEPLOY_VERSION__
```

Your branch is for 5.0 yet it is 4.4.2 here. Which?

Reply:
Since the original 5.0.1 security patch was duplicated in the 4.4 branch, I expect this one to be backported there as well, so 4.4.2 is listed there as the minimum version where this new method is expected.